### PR TITLE
Ali j branch2: Created scripts for automatically starting up Grafana Server and getting secrets from AWS (Grafana Key)

### PR DESCRIPTION
**Pull request by Ali-J2**
- Added aws_secrets.py: Used for fetching AWS secrets from AWS Secrets Manager; for use when fetching Grafana API Key
- Added ec2_instance_manager.py: Used for starting/stopping instances based on tag; for use with Grafana server which will be started if stopped.
- Modified create_stack_dash.py to include the aforementioned scripts in workflow.
- Modified create_stack: The Config object has been now includes new fields for retrieved Grafana secret from AWS and EC2 machine tag (for the EC2 instance that will automatically be started up with the Grafana instance)
- Instructions for create_stack_dash have been modified to include the new options available, sample env file depicted within also reflects these changes

### CODE REVIEW

**Review by jpoolGW — COMMENTED** [review]
Looks okay, I have requested a fix to be committed here where parameters in StartExecution.sh no longer have there values in quotes. Tis is preventing execution from running properly. I will approve when this fix is in

File: jmeter-icap-poc/scripts/config.env.sample [hunk 1]
```diff
@@ -4,7 +4,7 @@ SCRIPT_BUCKET=aws-testengine-s3
 SCRIPT_NAME=script/StartExecution.sh
 TEST_DATA_BUCKET=icap-perf-test-data
 TEST_DATA_FILE=files.csv
-SECRET_ID=GlasswallDataRepositoryTestUser
+SECRET_ID="GlasswallDataRepositoryTestUser
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/config.env.sample:7]: Quote left over

File: jmeter-icap-poc/scripts/create_stack_dash.py [hunk 2]
```diff
@@ -210,7 +189,7 @@ def main(config):
     Config.total_users = int(args.total_users)
     Config.users_per_instance = int(args.users_per_instance)
     Config.instances_required, Config.users_per_instance = __calculate_instances_required(Config.total_users,
-                                                               Config.users_per_instance)
+                                                                                          Config.users_per_instance)
```
Comment (jpoolGW) [inline jmeter-icap-poc/scripts/create_stack_dash.py:192]: Can this white space be removed?

**Review by jpoolGW — CHANGES_REQUESTED** [review]
2 minor changes needed

File: jmeter-icap-poc/scripts/config.env.sample (hunk 1 above)
Comment (Ali-J2) [reply jmeter-icap-poc/scripts/config.env.sample]: Alright, these should be good now

**Review by jpoolGW — APPROVED** [review]

**Pull request merged**